forge-ai/src/main/java/forge/ai/ComputerUtilCombat.java

```diff
@@ -2046,6 +2046,12 @@ public static Map<Card, Integer> distributeAIDamage(final Player self, final Car
             // TODO sort remaining tramplers for DamageDone triggers
         }
 
+        // Order the combatants in preferred order in case legacy ordering is disabled
+        if (!self.getGame().getRules().hasOrderCombatants()) {
+            // TODO: sometimes, orderAttackers needs to be called instead - differentiate this here somehow
```

Comment:
I have a feeling the names of `attacker` and `block` are misnomers here, so we should be determining if the params are attacking or blocking

```suggestion
            if (combatant.isAttacking()) { 
                opposedCombatants = AiBlockController.orderBlockers(combatant, new CardCollection(opposedCombatants));
            } else {
                 opposedCombatants = AiBlockController.orderAttackers(combatant, opposedCombatants);
            }
```